CODE_OF_CONDUCT.md

```diff
@@ -68,7 +68,7 @@ members of the project's leadership.
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+available at https://www.contributor-covenant.org/version/2/1/code_of_conduct/
```

Comment:
You have to copy-paste the file from the link not just update the link 😅 https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.md

Reply:
sorry, i will fix it